layout/theme.liquid

```diff
@@ -180,41 +260,52 @@
           will_be_in_stock_after: `{{ 'products.product.inventory.will_be_in_stock_after' | t: date: '[date]' }}`,
           waiting_for_stock: `{{ 'products.product.inventory.waiting_for_stock' | t }}`,
           out_of_stock: `{{ 'products.product.inventory.out_of_stock' | t }}`,
-          out_of_stock_continue_selling: `{{ 'products.product.inventory.out_of_stock_continue_selling' | t }}`,
-        },
+          out_of_stock_continue_selling: `{{ 'products.product.inventory.out_of_stock_continue_selling' | t }}`
+        }
       };
 
       window.accessibilityStrings = {
         imageAvailable: `{{ 'products.product.media.image_available' | t: index: '[index]' }}`,
         shareSuccess: `{{ 'general.share.success_message' | t }}`,
         pauseSlideshow: `{{ 'sections.slideshow.pause_slideshow' | t }}`,
-        playSlideshow: `{{ 'sections.slideshow.play_slideshow' | t }}`,
+        playSlideshow: `{{ 'sections.slideshow.play_slideshow' | t }}`
       };
 
       window.themeStrings = {
         product_count_simple_one: `{{ 'products.facets.product_count_simple.one' | t: count: '[count]' }}`,
         product_count_simple_other: `{{ 'products.facets.product_count_simple.other' | t: count: '[count]' }}`,
         results_with_count_one: `{{ 'templates.search.results_with_count.one' | t: count: '[count]' }}`,
-        results_with_count_other: `{{ 'templates.search.results_with_count.other' | t: count: '[count]' }}`,
+        results_with_count_other: `{{ 'templates.search.results_with_count.other' | t: count: '[count]' }}`
       };
 
       theme = {};
       theme.settings = {
-        moneyFormat: {{ shop.money_format | json }},
+        moneyFormat: {{ shop.money_format | json }}
       };
 
       theme.assets = {};
     </script>
 
-    <svg xmlns="http://www.w3.org/2000/svg" width="0" height="0">
```

Comment:
Hello @Akeron44 , Where we are using the SVG that you have added on this file?

Reply:
It was part of the theme, not added by me. Should I remove it from theme.liquid file?